index.js

```diff
@@ -105,7 +107,10 @@ function fastifyStatic (fastify, opts, next) {
 
   if (opts.decorateReply !== false) {
     fastify.decorateReply('sendFile', function (filePath) {
-      pumpSendToReply(this.request, this, filePath)
+      // only resolve to avoid unhandled rejections
+      return new Promise((resolve, reject) => {
+        pumpSendToReply(this.request, this, filePath, () => resolve())
```

Comment:
If the user use this signature:
```
function (req, reply) {
   return reply.sendFile('/index-not-exists.html')
}
```

I have afraid we could have wrong logging as a side effect

```suggestion
       pumpSendToReply(this.request, this, filePath, (err) => {
         if (err) { reject(err) } else { resolve() }
       })
```

Reply:
Good catch, in that case we should document that `sendFile` returns always undefined and then the default fastify promise resolution will apply.

Reply:
Your suggestion can lead to an unhandlef rejection that's why I use resolve only. The value isn't important only the control flow.

Reply:
Can you add a comment on this one?